lib/util/fs/index.ts

```diff
@@ -38,7 +39,7 @@ export async function readLocalFile(
     const fileContent = encoding
       ? await fs.readFile(localFileName, encoding)
       : await fs.readFile(localFileName);
-    return fileContent;
+    return fileContent.toString().replace(regEx(/\r\n/g), '\n');
```

Comment:
NO, don't change this file. This should be only be done in maven manager, but normally is shouldn't necessary at all.

Reply:
you mean gradle right?  this is gradle task

Reply:
`gradle/extract.ts` sounds good?